index.html

```diff
@@ -0,0 +1,2199 @@
+              <div class="image_small">
+                <img
+                  src="https://d.newsweek.com/en/full/1581194/zoom.jpg?w=95&amp;h=95&amp;f=5481146166f015d5f2f5380625324995"
+                  alt="a" />
+              </div>
+              <div class="story_info_container flex-column pl-3">
+                <h2 >
+                  <a class="title" href="#">
+                    More Than 500,000 Zoom Account Credentials Being Sold on Dark
+                    Web
+                  </a>
+                </h2>
+                <div class="paragraph"></div>
+              </div>
+            </article>
+          </div>
+        </div>
+      </div>
+  
+      <div class="row fourth-section-wrapper border-bottom">
+        <div class=" col-md-12 col-lg-8 sub-wrapper">
+          <section>
+            <div class=" col-lg-8 d-flex align-items-center justify-content-start font-roboto sub-wrapper2">
+              <h2 class="font-weight-bold sub-title">SUBSCRIBE</h2>
+            </div>
+            <article class="w-100 sub-container">
+              <div class="d-flex">
+                <div>
+                  <img class="sub-img w-100 " src="https://d.newsweek.com/en/full/1579878/april-17-24-2020-cover.jpg" alt="subscribe-img">
+                </div>
+                <div class="d-flex flex-column align-items-start pt-5 sub-paragraph">
+                  <h2 class="mag_link2 py-2">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h2>
+                  <p class="py-2">All Access + Weekly Delivery.
+                    Choose A Membership That's Perfect for You!</p>
+                  <div class="py-2">
+                      <button type="button" class="subscribe_btn">SUBSCRIBE ></button>
+                  </div>
+                </div>
+              </div>
+            </article>
+          </section>
+        </div>
+        <div class="col-md-12 col-lg-4 signup-wrapper">
+          <div class=" col-lg-4 d-flex align-items-center justify-content-start font-roboto">
+            <h2 class="font-weight-bold sub-title">NEWSLETTER</h2>
+          </div>
+          <article class="w-100  sign-up-container">
+            <div class="d-flex">
+              <div>
+                <img class="sign-up-img w-100" src="https://d.newsweek.com/en/full/1579878/april-17-24-2020-cover.jpg" alt="subscribe-img">
+              </div>
+              <div class="d-flex flex-column align-items-start sub-paragraph">
+                <h2 class="mag_link2 py-2">The Starting 5</h2>
+                <p class="py-2">See why nearly a quarter of a million subscribers
+                   begin their day with the Starting 5.
+                </p>
+                  <div class="py-2">
+                    <button type="button" class="subscribe_btn">SIGN UP ></button>
+                  </div>                
+              </div>
+            </div>
+          </article>
+
+        </div>
+    
```

Comment:
This part doesn't look like the original page, yet. Take a look at the Subscribe div: it has two images together.
The Newsletter div has a different image. The titles of both have different font-size...
Yours:
![Screenshot from 2020-04-22 13-25-19](https://user-images.githubusercontent.com/51926937/80019300-edf14c80-849c-11ea-9bfd-0d3a52153cf6.png)

Original:
![Screenshot from 2020-04-22 13-25-27](https://user-images.githubusercontent.com/51926937/80019312-f184d380-849c-11ea-90cf-2b2842cf4d74.png)


On mobile, the section behaves differently on your page. Please make it look like the original.
Yours:
![Screenshot from 2020-04-22 13-38-53](https://user-images.githubusercontent.com/51926937/80020490-c307f800-849e-11ea-9855-bcd6b7e29332.png)

Original:
![Screenshot from 2020-04-22 13-39-01](https://user-images.githubusercontent.com/51926937/80020502-c69b7f00-849e-11ea-9083-474efc6808e7.png)